src/util/native_fs_utils.js

```diff
@@ -433,13 +433,14 @@ function isDirectory(ent) {
 
 /**
  * @param {string} [config_root_backend]
+ * @param {number} [warn_threshold_ms]
  * @returns {nb.NativeFSContext}
  */
-function get_process_fs_context(config_root_backend) {
+function get_process_fs_context(config_root_backend, warn_threshold_ms) {
     return {
         uid: process.getuid(),
         gid: process.getgid(),
-        warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
+        warn_threshold_ms: warn_threshold_ms || config.NSFS_WARN_THRESHOLD_MS,
         backend: config_root_backend
```

Comment:
The JSDoc indicates (by using the square brackets) that the value `config_root_backend` is optional and we have a default value in the config file, so I suggest using it.
```suggestion
        backend: config_root_backend || config.NSFS_NC_CONFIG_DIR_BACKEND,
```
Then your calls to `get_process_fs_context('', 100)` would change to `get_process_fs_context()` (using the default values).

Reply:
get_process_fs_context() is being called from various files, not in all of them config.NSFS_NC_CONFIG_DIR_BACKEND is the default, I changed the parameter name from config_root_backend to backend so it'll be more understood

Reply:
What about changing your calls from `get_process_fs_context('', 100)` to `get_process_fs_context()` (using the default values)?

Reply:
done, thanks!